src/Sylius/Bundle/PromotionBundle/Resources/config/services.xml

```diff
@@ -145,11 +147,19 @@
             <argument type="service" id="sylius.factory.promotion_coupon" />
             <argument type="service" id="sylius.repository.promotion_coupon" />
             <argument type="service" id="sylius.manager.promotion_coupon" />
+            <argument type="service" id="sylius.generator.percentage_policy" />
+        </service>
+        <service id="sylius.generator.percentage_policy" class="%sylius.generator.percentage_policy.class%">
```

Comment:
This is specific to promotion coupons, not a generic generation policy for whole Sylius: ``sylius.generator.promotion_coupon.percentage_policy`` or something like that.